doc/040_backup.rst

```diff
@@ -187,6 +187,23 @@ On **Windows**, a file is considered unchanged when its path and modification
 time match, and only ``--force`` has any effect. The other options are
 recognized but ignored.
 
+Dry Runs
+********
+
+You can perform a backup in dry run mode to see what would happen without
+modifying the repo.
+
+-  ``--dry-run``/``-n`` do not write anything, just print what would be done
+
+Combined with ``--verbose``, you can see a list of changes:
+
+.. code-block:: console
+
+    $ restic -r /srv/restic-repo backup ~/work --dry-run -vv | grep "added)$"
```

Comment:
you say that `--dry-run` has to be combined with `--verbose`, yet you combine it with `-vv`

Reply:
`-vv` equals to `--verbose=2`